data/mods/Magiclysm/enchanted/enchanted_tools.json

```diff
@@ -0,0 +1,66 @@
+[
+  {
+    "id": "copper_bracelet_manafocus_off",
+    "type": "ARMOR",
+    "name": "copper mana focusing bracelet",
+    "description": "An ornate copper bracelet covered in magical symbols, crafted and manipulated to act as a focus for magical powers when you are enchanting. This is the second most basic enchanting tool. Activate to say the command word and turn the bracelet into a pencil like tool with a sharp point that you can use to etch magical symbols into other materials.",
```

Comment:
there's no need too say in the description of the item how basic of a tool it is, that's self-evident by the qualities it possesses.

Reply:
True enough, but people have complained that we don't explain how magiclysm works, and there's no guidebook for it, in or out of game, so I was trying to give people an understanding of how the process works, why it takes so much time, etc.  It's flavor text.  Besides, there's no determination as of yet how many levels of tools there will be. I assume at least one more rank of them, but a player has no way of knowing that there might be others.

Reply:
i think books and survivor notes and snippets might help with this.

Reply:
I had thought about writing a basic note to put in their inventory on exactly that subject, but non-mage starts wouldn't necessarily find that, they'd stumble across the tool and have no idea what it was for.

Reply:
i dunno about you, but if i found a random circlet that turns into a rod and i wasn't a trained mage, i wouldn't know what it was for.

Reply:
We haven't exactly discussed how that's all going to be accomplished, you didn't want a spell to do enchanting, but do we need recipe books? I don't know how to kickstart a new crafting system, I had to use Shard's work as a template to get the menu item as it was.

Reply:
> 
> 
> yes, the descriptive text, and some of the plural names. i didn't remember that conversation, sorry.

Not a problem, I'm aware of them, but it seemed better to push it out rather than me screw around with it. I'm aware of those and the plural issues, just didn't get to it yet.

Reply:
> We haven't exactly discussed how that's all going to be accomplished, you didn't want a spell to do enchanting, but do we need recipe books? I don't know how to kickstart a new crafting system, I had to use Shard's work as a template to get the menu item as it was.

recipe books yes. i think using the ingame crafting system is a reasonable way of doing it.

Reply:
Well, we should probably have some kind of meeting to figure out how we want to break that up, because I've just started scratching the surface on items, so far with no recipes, we're just building up the loot and infrastructure for it.